packages/mdc-menu/simple/adapter.js

```diff
@@ -163,8 +163,11 @@ class MDCSimpleMenuAdapter {
   * }} position */
   setPosition(position) {}
 
+  /** @param {string} height */
+  setMaxHeight(height) {}
```

Comment:
remember....before you hit the "squash and merge button" to add a "breaking change" comment to the commit.

Reply:
So adding, a method is considered a breaking change?

Reply:
When it comes to adapter methods, any change is considered breaking, since it impacts authors of libraries which wrap our components (since they have to implement the adapter API themselves).